src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -54,6 +54,7 @@ impl Step for CrateBootstrap {
         run.path("src/tools/jsondoclint")
             .path("src/tools/suggest-tests")
             .path("src/tools/replace-version-placeholder")
+            .path(check::CoverageDump::PATH)
```

Comment:
nit: I'm not sure about putting the paths and display names in constants, since it's inconsistent with the other entries here and makes it a little hard to discover at a glance.
Up to you if the advantages here outweigh the downsides

Reply:
It seemed like a good idea at the time, but I think I'll go back to hard-coded paths, since there isn't a big practical advantage to using a constant here.